go.mod

```diff
@@ -158,29 +156,35 @@ require (
 	k8s.io/component-base v0.22.5 // indirect
 	k8s.io/gengo v0.0.0-20211129171323-c02415ce4185 // indirect
 	k8s.io/klog/v2 v2.40.1 // indirect
-	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65 // indirect
-	k8s.io/utils v0.0.0-20211208161948-7d6a63dca704 // indirect
-	knative.dev/caching v0.0.0-20220223181140-b3aed5c4e283 // indirect
+	k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf // indirect
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
+	knative.dev/caching v0.0.0-20220311005831-b2a3f59d7175 // indirect
 	knative.dev/reconciler-test v0.0.0-20220118183433-c8bfbe66bada
-	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
 replace (
 	// Knative forks.
-	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20220427101530-5a779a5c3f55
-	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20220427170443-321d4a2b113a
-	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20220422063503-04b2eff6d979
-	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20220421145848-cbcc812ef96f
+	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20220602061606-48e3ac9b02e8
+	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20220505101922-57970a157b14
+	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20220602134621-9e54bd6ff698
+	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20220603141301-7e612b35a9b3
 )
 
 replace (
 	// Adjustments to align transitive deps
 	github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
 	github.com/go-logr/zapr => github.com/go-logr/zapr v0.4.0
 	github.com/manifestival/manifestival => github.com/manifestival/manifestival v0.7.0
+
+	k8s.io/api v0.23.4 => k8s.io/api v0.22.5
+	k8s.io/apimachinery v0.23.4 => k8s.io/apimachinery v0.22.5
+	k8s.io/client-go v0.23.4 => k8s.io/client-go v0.22.5
```

Comment:
The `k8s.io` replacements probably can be eliminated if we bump Eventing too.

Reply:
I'd rather bump every component dependencies in one go.